src/main/java/com/teamwizardry/shotgunsandglitter/common/core/MiniTurretHelper.java

```diff
@@ -0,0 +1,34 @@
+package com.teamwizardry.shotgunsandglitter.common.core;
+
+import com.teamwizardry.librarianlib.features.config.ConfigProperty;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityList;
+import net.minecraft.util.ResourceLocation;
+
+import java.util.HashSet;
+
+public class MiniTurretHelper {
+    @ConfigProperty(category = "turret", comment = "Add the mobs below which are exempt from being killed by the turret")
+    public static String[] blacklistedEntityName = {"cow", "sheep", "pig", "chicken", "rabbit"};
```

Comment:
The blacklist should be empty by default, with a separate "don't target passives" check